src/util.js

```diff
@@ -19,14 +20,15 @@ exports = module.exports
 /**
  * Get the CID of the DAG-Node.
  *
- * @param {Object} dagNode - Internal representation
+ * @param {Buffer} blob - Serialized binary data
```

Comment:
I would change it to `Encoded Protocol Buffers blob` or `Serialized as Protocol Buffers`.